src/luxos/scripts/async_luxos.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+import asyncio
+import json
+
+from .. import misc
+from .. import asyncops
+
+
+async def run(
+    ip_list: list[str], port: int, cmd: str, params: list[str], timeout: float,
+    delay: float | None, details: bool
+) -> None:
+
+    batchsize = 50
```

Comment:
Maybe we can make this value something that could be changed through the config file (or any other dynamic approach)

Reply:
done, it now leverages the `--max_threads`, set by default to 10